build.sbt

```diff
@@ -28,18 +27,20 @@ lazy val math = crossProject(JVMPlatform, JSPlatform)
       "org.typelevel"              %%% "cats-core"               % catsVersion,
       "com.github.julien-truffaut" %%% "monocle-core"            % monocleVersion,
       "com.github.julien-truffaut" %%% "monocle-macro"           % monocleVersion,
-      "org.scala-lang.modules"     %%% "scala-collection-compat" % collCompatVersion
+      "org.scala-lang.modules"     %%% "scala-collection-compat" % collCompatVersion,
+      "edu.gemini"                 %%% "gpp-jts"                 % jtsVersion
     )
   )
   .jvmConfigure(_.enablePlugins(AutomateHeaderPlugin))
   .jvmSettings(
     libraryDependencies ++= Seq(
-      "org.locationtech.jts" % "jts-core" % jtsVersion
+      "edu.gemini"                 %%% "gpp-jts-awt"             % jtsVersion
     )
   )
   .jsSettings(gspScalaJsSettings: _*)
   .jsSettings(
-    libraryDependencies += "io.github.cquiroz" %%% "scala-java-time" % scalaJavaTimeVersion
+    libraryDependencies += "io.github.cquiroz" %%% "scala-java-time" % scalaJavaTimeVersion,
+    scalaJSUseMainModuleInitializer := true,
```

Comment:
What do we do with this setting?

Reply:
We probably don't need it. It lets us build a runnable example on js, equivalent to JTSDemo but if we move the "demo" code to another module we can remove it